src/nls/root/strings.js

```diff
@@ -138,6 +138,7 @@ define({
     "EXT_MODIFIED_WARNING"              : "<span class='dialog-filename'>{0}</span> has been modified on disk outside of {APP_NAME}.<br /><br />Do you want to save the file and overwrite those changes?",
     "EXT_MODIFIED_MESSAGE"              : "<span class='dialog-filename'>{0}</span> has been modified on disk outside of {APP_NAME}, but also has unsaved changes in {APP_NAME}.<br /><br />Which version do you want to keep?",
     "EXT_DELETED_MESSAGE"               : "<span class='dialog-filename'>{0}</span> has been deleted on disk outside of {APP_NAME}, but has unsaved changes in {APP_NAME}.<br /><br />Do you want to keep your changes?",
+    "WINDOW_UNLOAD_WARNING"             : "You have unsaved changes! Are you sure you want to quit Brackets and navigate to a different URL?",
```

Comment:
The message looks incorrect. Quit brackets and navigate? Should you just have are you sure you want to quit brackets? or your unsaved changes will be lost?

Reply:
Brackets is just an app which was being run in shell. Think of this as having an unpaid cart and try to move away from there by typing an URL in the address bar.

Reply:
Do you want to rephrase it as "Are you sure you want to navigate to a different URL and quit Brackets?"

Reply:
Makes sense. I will change the message.